sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/implementation/models/PhoneticTokenFilter.java

```diff
@@ -33,7 +33,7 @@
      * tokens. If false, encoded tokens are added as synonyms. Default is true.
      */
     @JsonProperty(value = "replace")
-    private Boolean replaceOriginalTokens;
+    private Boolean originalTokensReplaced;
```

Comment:
Is this a generated class? It's in implementation models but I know many of these classes were copied from generated code.

Reply:
Good catch. Reverts all changes under impl